llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2033,6 +2033,8 @@ class VPReductionPHIRecipe : public VPHeaderPHIRecipe,
   /// Generate the phi/select nodes.
   void execute(VPTransformState &State) override;
 
+  unsigned getVFScaleFactor() const { return VFScaleFactor; }
```

Comment:
Perhaps good to have comments on both new functions added?

Reply:
Done.